package.json

```diff
@@ -8,6 +8,7 @@
     "@lit/localize": "^0.12.1",
     "@shoelace-style/shoelace": "^2.12.0",
     "autonumeric": "^4.8.1",
+    "clsx": "^2.1.0",
```

Comment:
Good idea. I think this might clean up some of the lists/joins that you're doing on other PRs too?

Reply:
Yeah, I think I'll go back and clean that up after the bulk of the work is done.